uber/automated_emails.py

```diff
@@ -60,7 +60,7 @@ def send_all(cls, raise_errors=False):
                 attendees, groups = session.everyone()
                 approved = {ae.subject for ae in session.query(ApprovedEmail).all()}
                 models = {Attendee: attendees, Group: groups}
-                models.update({model: lister(session) for model, lister in self.extra_models.items()})
+                models.update({model: lister(session) for model, lister in cls.extra_models.items()})
```

Comment:
This is a classmethod, so we need to say ``cls`` instead of ``self``.

Rookie mistake, me.